### PR TITLE
Milestone 7: feature/Add forms to blog

### PR DESCRIPTION
**Pull request by brainconnect93**
- Added web forms and crud method for creating a post
- Added web forms and crud method for creating a comment
- Added web forms and crud method for adding a like to a post
- Add navigation links to all the forms created

> Kindly review and approve

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+  def create
+    @post = current_user.posts.new(post_params[:id])
+    respond_to do |format|
+      format.html do
+        if @post_new.save
+          redirect_to "/users/#{@post_new.user_id}/posts/", message: 'Post created successfully'
+        else
+          render :new, status: 'Error: Post not created'
+        end
+      end
```
Comment (AshakaE) [inline app/controllers/posts_controller.rb]: `@post_new` has no reference
- [ ] Kindly make sure `@post_new` is declared variable

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @brainconnect93  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Git flow ✅ 
- Pull request ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (brainconnect93) [reply app/controllers/posts_controller.rb]: @AshakaE Thanks i will look into that now and make necessary changes

**Review by BenMukebo — APPROVED** [review]
### Hi @brainconnect93, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- All necessary controllers, views and routers are present and well implemented 🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brainconnect93** [comment]
@BenMukebo Thanks for the approval

**Pull request merged**